### DISCUSSION TITLE
Hello friends. I am new here . I will apply, but I am waiting for my student profile to be verified. Is there anyone who can help me with the application and the progress of this system? Thank you for your interest.

### DISCUSSION THREAD

**Discussion by Fizikci22** [community/community#56843] [GitHub Education]
### Hi everyone,

Hello friends. I am new here . I will apply, but I am waiting for my student profile to be verified. Is there anyone who can help me with the application and the progress of this system? Thank you for your interest.

**Comment by Ntshangase** [comment]
Hey @Fizikci22  you have been verified and can start enjoying your student benefits🎉

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
